apps/wing/src/commands/lsp.ts

```diff
@@ -144,11 +144,28 @@ export async function lsp() {
           }))
         );
 
+        // Add annotations as notes hinting back to the original diagnostic
+        const extraNotes = rd.annotations.map((a) =>
+          Diagnostic.create(
+            Range.create(a.span.start.line, a.span.start.col, a.span.end.line, a.span.end.col),
+            a.message,
+            DiagnosticSeverity.Hint,
+            undefined,
+            undefined,
+            [
+              {
+                location: Location.create(diagnosticUri, diag.range),
+                message: `(Source) ${diag.message}`,
```

Comment:
Feels like this doesn't have to be caps but I'm not strongly opinionated on it
```suggestion
                message: `(source) ${diag.message}`,
```